HelpSource/Classes/Routine.schelp

```diff
@@ -66,11 +66,12 @@ passing the code::inval:: argument as the return value of code::yield::.
 link::#-stop:: has been called).
 ::
 
-Since Routine inherits from link::Classes/Thread::, it will become the
+code::thisThread:: : Since Routine inherits from link::Classes/Thread::, it will become the
 emphasis::current thread:: when it is started or resumed; i.e.
 link::Classes/Thread#.thisThread#thisThread:: used in the Routine Function will return
-the Routine. It will inherit the parent thread's logical time and clock
-(see link::Classes/Thread#-parent::).
+the Routine.
+
+Time: Just before code::next:: is called, code::thisThread:: points either to another Routine, or the top-level Thread. During the code::next:: call, this becomes the "parent thread" (see link::Classes/Thread#-parent::). code::next:: then evaluates on the parent's clock, at the parent's logical time.
```

Comment:
scare quotes unnecessary

Reply:
also, 'at the parent's logical time' or 'on the parent's logical time'?

Reply:
OK, I'll remove the quotes -- but I don't agree with the characterization as *scare* quotes. In technical writing, sometimes you have to introduce a phrase with a specific meaning that may not be obvious. It's fairly standard to highlight that in some way, either by *emphasis* or "quoting." (Or perhaps times have changed and quote marks are always "ironic" now.)

Definitely *at* a specific time.

I think I took care of all the changes now (hope I didn't miss one).

Reply:
"scare quotes" also denote non-standard usage, while all of these usages (as far as i can tell) are consistent with their standard definition